cmd/kubeadm/app/cmd/init.go

```diff
@@ -42,20 +44,25 @@ import (
 )
 
 var (
-	initDoneMsgf = dedent.Dedent(`
+	initDoneTempl = template.Must(template.New("init").Parse(dedent.Dedent(`
 		Your Kubernetes master has initialized successfully!
 
-		To start using your cluster, you need to run:
-		export KUBECONFIG=%s
+		To start using your cluster, you need to run (as a regular user):
+
+		  sudo cp {{.KubeConfigPath}} $HOME/
```

Comment:
I think this is much clearer and a step in the correct direction of making docs for kubeadm better. My only comment is about making kubeadm as OS-agnostic as much as possible (https://github.com/kubernetes/kubeadm/issues/108). Obviously, kubeadm is not supported for windows or macOS yet. Maybe writing in plain English what you are doing (in these commands) and saying something like:
"For example, run these commands on your linux machine."

Just an idea.